interpreter/runtime/instance.ml

```diff
@@ -43,6 +43,13 @@ let () =
     | FuncRef _ -> "func"
     | r -> string_of_ref' r
 
+let () =
```

Comment:
Is this an intended change?

Reply:
Ah, yes: it is related to the refactoring of result matching. While doing that, I noticed that the way it compared references was actually bogus, and a proper function is needed for that. I could have fixed that separately, but while I was here...

Reply:
Got it, LGTM!